controller/src/main/java/gov/va/vro/controller/VroController.java

```diff
@@ -9,11 +9,9 @@
 import gov.va.vro.api.requests.HealthDataAssessmentRequest;
 import gov.va.vro.api.resources.VroResource;
 import gov.va.vro.api.responses.ClaimMetricsResponse;
```

Comment:
If you just want to disable some endpoints in some environment, should you just use a feature flag to disable/enable them? This looks too complicated if the task is as the Jira ticket MCP-1962 says.